.golangci.yaml

```diff
@@ -7,6 +7,7 @@ issues:
     - "should not use dot imports"
     - "don't use an underscore in package name"
     - "exported: .*"
+    - "\"k8s.io/utils/pointer\" is deprecated"
```

Comment:
why?

Reply:
The `kubedog` library depends on the new version of the `k8s.io/utils/pointer` package, in the new version some functions are marked as deprecated, we use these functions in more than 100 files.